src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -283,14 +282,6 @@ def dropna():
              name        toy       born
         1  Batman  Batmobile 1940-04-25
 
-        Drop the columns where at least one element is missing.
-
```

Comment:
Why does this not work anymore?

Reply:
dropna with axis=1 isn't supported

https://github.com/snowflakedb/snowpark-python/blob/75cba1001fd7d09c40ea76a67a0320176cfc4942/src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py#L7970